### PR TITLE
Add whisper endpoints

### PR DESCRIPTION
**Pull request by Clemalfroy**
Implementation of the 2 endpoints for OpenAi [Whisper](https://platform.openai.com/docs/guides/speech-to-text/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.5.0] - 2023-03-02
+
+### Added
+
+- Add Client#transcribe endpoint - Whisper transcription over the wire!
+- Add Client#translate endpoint - Whisper translation over the wire!
```
Comment (alexrudall) [inline CHANGELOG.md:13]:
```suggestion
- Add Client#transcribe and Client translate endpoints - Whisper over the wire! Thanks to [@Clemalfroy](https://github.com/Clemalfroy)

```

File: README.md [hunk 2]
```diff
@@ -267,6 +267,40 @@ Pass a string to check if it violates OpenAI's Content Policy:
     => 5.505014632944949e-05
 ```
 
+### Whisper
+
+Whispr is a speech to text model that can be used to generate text in a conversational style. You can use it to generate a text based on a audio file [messages](https://platform.openai.com/docs/guides/chat/introduction):
+
+
```
Comment (alexrudall) [inline README.md:274]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -267,6 +267,40 @@ Pass a string to check if it violates OpenAI's Content Policy:
     => 5.505014632944949e-05
 ```
 
+### Whisper
+
+Whispr is a speech to text model that can be used to generate text in a conversational style. You can use it to generate a text based on a audio file [messages](https://platform.openai.com/docs/guides/chat/introduction):
```
Comment (alexrudall) [inline README.md:272]:
```suggestion
Whisper is a speech to text model that can be used to generate text based on an audio file [messages](https://platform.openai.com/docs/guides/chat/introduction):
```

File: README.md [hunk 4]
```diff
@@ -267,6 +267,40 @@ Pass a string to check if it violates OpenAI's Content Policy:
...
+
+
+#### Translate
+
+The translations API takes as input the audio file in any of the supported languages and transcribes, if necessary, the audio into english
+
+
```
Comment (alexrudall) [inline README.md:279]:
```suggestion
```

**Review by alexrudall — CHANGES_REQUESTED** [review]
Looking good! Few tweaks and I'll merge

**Review by alexrudall — APPROVED** [review]

**Comment by alexrudall** [comment]
Thanks a lot @Clemalfroy! Nice work

**Comment by richardonrails** [comment]
Cool! This sends the local audio file to OpenAI? If you already have a file on S3 do you need to download it first?

**Pull request merged**